lib/runner.js

```diff
@@ -461,6 +461,36 @@ internals.executeTests = function (experiment, state, skip, callback) {
 };
 
 
+internals.experimentHasTests = function (experiment, state) {
```

Comment:
Shall use .some() instead of filters in this, there's no need to have a full count, just need at least one.